bundles/org.openhab.binding.modbus/src/main/resources/ESH-INF/thing/bridge-tcp.xml

```diff
@@ -25,40 +25,40 @@
 			</parameter>
 			
 			<parameter name="enableDiscovery" type="boolean">
-				<label>Discovery enabled</label>
+				<label>Discovery Enabled</label>
 				<description>When enabled we try to find a device specific handler. Turn this on if you're using one of the supported devices.</description>
 				<default>false</default>
 			</parameter>
 
 			<!-- connection handling -->
 			<parameter name="timeBetweenTransactionsMillis" type="integer" min="0">
-				<label>Time between transactions (ms)</label>
+				<label>Time Between Transactions (ms)</label>
 				<description>How long to delay we must have at minimum between two consecutive MODBUS transactions. In milliseconds.
 				</description>
 				<default>60</default>
 			</parameter>
 			<parameter name="timeBetweenReconnectMillis" type="integer" min="0">
-				<label>Time between reconnections (ms)</label>
+				<label>Time Between Reconnections (ms)</label>
 				<description>How long to wait to before trying to establish a new connection after the previous one has been
 					disconnected. In milliseconds.</description>
 				<default>0</default>
 				<advanced>true</advanced>
 			</parameter>
 			<parameter name="connectMaxTries" type="integer" min="1">
-				<label>Maximum connection tries</label>
+				<label>Maximum Connection Tries</label>
 				<description>How many times we try to establish the connection. Should be at least 1.</description>
 				<default>1</default>
 				<advanced>true</advanced>
 			</parameter>
 			<parameter name="reconnectAfterMillis" type="integer" min="0">
-				<label>Disconnect and establish the connection again after this time (ms)</label>
+				<label>Disconnect and Establish the Connection Again After This Time (ms)</label>
```

Comment:
That's a good example why we should think about a strict length limit for labels :-)

Reply:
I already added such a check to the new label SAT check I'm implementing. The question is what would be a good length limit?